modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java

```diff
@@ -15,14 +15,401 @@
+import com.liferay.headless.admin.workflow.resource.v1_0.test.util.WorkflowInstanceTestUtil;
+import com.liferay.headless.admin.workflow.resource.v1_0.test.util.WorkflowTaskTestUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.model.Role;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.model.role.RoleConstants;
+import com.liferay.portal.kernel.resource.bundle.ResourceBundleLoader;
+import com.liferay.portal.kernel.resource.bundle.ResourceBundleLoaderUtil;
+import com.liferay.portal.kernel.security.permission.PermissionCheckerFactoryUtil;
+import com.liferay.portal.kernel.security.permission.PermissionThreadLocal;
+import com.liferay.portal.kernel.service.RoleLocalService;
+import com.liferay.portal.kernel.service.UserLocalServiceUtil;
+import com.liferay.portal.kernel.test.rule.DataGuard;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.workflow.WorkflowTaskManager;
+import com.liferay.portal.test.rule.Inject;
 
-import org.junit.Ignore;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 
 /**
  * @author Javier Gamarra
  */
-@Ignore
+@DataGuard(scope = DataGuard.Scope.METHOD)
 @RunWith(Arquillian.class)
 public class WorkflowLogResourceTest extends BaseWorkflowLogResourceTestCase {
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		BaseWorkflowLogResourceTestCase.setUpClass();
+
+		_resourceBundleLoader =
+			ResourceBundleLoaderUtil.
+				getResourceBundleLoaderByBundleSymbolicName(
+					"com.liferay.portal.language.lang");
+
+		_workflowDefinition =
+			WorkflowDefinitionTestUtil.addWorkflowDefinition();
+
+		PermissionThreadLocal.setPermissionChecker(
+			PermissionCheckerFactoryUtil.create(
+				UserLocalServiceUtil.getUser(TestPropsValues.getUserId())));
+	}
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		BaseWorkflowLogResourceTestCase.setUpClass();
+
+		_siteContentReviewerRole = _roleLocalService.getRole(
+			testGroup.getCompanyId(), RoleConstants.SITE_CONTENT_REVIEWER);
+
+		_workflowInstance = WorkflowInstanceTestUtil.addWorkflowInstance(
+			testGroup.getGroupId(), ObjectReviewedTestUtil.addObjectReviewed(),
+			_workflowDefinition);
+	}
+
+	@Override
+	@Test
+	public void testGetWorkflowInstanceWorkflowLogsPage() throws Exception {
+		WorkflowTask workflowTask = WorkflowTaskTestUtil.getWorkflowTask(
+			_workflowInstance.getId());
+
+		User user = TestPropsValues.getUser();
+
+		Assignee assignee = AssigneeTestUtil.addAssignee(user.getGroup());
```

Comment:
The `AssigneeTestUtil.addAssignee` method creates an user, we need create this user or we can get this other one`TestPropsValues.getUser()`?